api-docs/getting-started/concepts.rst

```diff
@@ -22,7 +22,7 @@ Users can also be assigned to a tenant or region so that they inherit a set of
 access rights and privileges automatically, based on the tenant or region
 configuration.
 
-In the Rackspace Cloud Users are represented as accounts created for assigns a
+In the Rackspace Cloud, users are represented as accounts created for a
```

Comment:
Suggest "In the Rackspace Cloud, users are represented as accounts. The accounts are automatically created with a default user type and role, based on the Identity service system configuration. The user type and role determine the resources and capabilities that are available to the user."